wp/plugin/api/c/cbat_libc.h

```diff
@@ -0,0 +1,252 @@
+#include <stddef.h>
+#include "cbat.h"
+
+// Set LIBC_UNROLL to 4, 8, 12 or 16. Default is 12.
+#ifndef LIBC_UNROLL
+#define LIBC_UNROLL 12
+#endif
+
+#define strcmp cbat_libc_strcmp
+#define strlen cbat_libc_strlen
+#define strcpy cbat_libc_strcpy
+#define strncpy cbat_libc_strncpy
+#define memcmp cbat_libc_memcmp
+#define memcpy cbat_libc_memcpy
+
+#define UNROLL_EXCEEDED __VERIFIER_assume(true)
```

Comment:
Pretty sure we want this to be `__VERIFIER_assume(false)`, since `assume(true)` is a no-op :)

Reply:
Yeah, that's what I started with but it did not behave as expected. I'll show you on examples when we discuss next